openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/CollectionDAO.java

```diff
@@ -3592,6 +3592,32 @@ interface DataQualityDataTimeSeriesDAO extends EntityTimeSeriesDAO {
     default String getTimeSeriesTableName() {
       return "data_quality_data_time_series";
     }
+
+    @ConnectionAwareSqlQuery(
+        value =
+            "SELECT json FROM data_quality_data_time_series where entityFQNHash = :testCaseFQNHash "
```

Comment:
its worth promoting incidentId as field at top level and add a index insted of doing JSON_EXTRACT